specification/schema/serverless-workflow-schema.json

```diff
@@ -182,6 +182,12 @@
           "type": "string",
           "description": "CloudEvent type"
         },
+        "kind": {
+          "type" : "string",
+          "enum": ["consumed", "produced"],
```

Comment:
I was wondering - if it's a binary choice (either consumed or produced), then the presence or absence of the source field alone could determine the kind (that would require a documented specification but would not require the addition of a 'kind' field). Unless the kind can be both, in which case the source field is required yet it is also an event that could be produced.

Reply:
added comment on this below.